### PR TITLE
Zaid/hotfix

### PR DESCRIPTION
**Pull request by zaidokal**
# JIRA Ticket Link

N/A

# Summary of Changes

Hotfixes

### CODE REVIEW

**Review by arhamansarii — DISMISSED** [review]
LGTM!

File: src/app/components/CompanyOverlay.tsx [hunk 1]
```diff
@@ -20,20 +20,38 @@ interface CompanyOverlayProps {
...
+            ))}
+          </div>
+        ) : (
+          <p className="text-[24px]">No reasoning available</p>
+        )}
       </div>
+
+      <img
+        src="screenshots/footer.webp"
+        alt="Footer Image"
```
Comment (skhan943) [inline src/app/components/CompanyOverlay.tsx:52]: Can you make this alt text a bit more descriptive. i.e. Sustain Engine

**Review by rakhshankhan — CHANGES_REQUESTED** [review]
Can you please ensure that the hydration fix works

File: src/app/components/CompanyOverlay.tsx (hunk 1 above)
Comment (zaidokal) [reply src/app/components/CompanyOverlay.tsx]: DONE

**Review by skhan943 — APPROVED** [review]

**Review by arhamansarii — APPROVED** [review]
lgtm

**Review by rakhshankhan — APPROVED** [review]

**Pull request merged**